gm4/plugins/create_manifest.py

```diff
@@ -0,0 +1,74 @@
+from beet import Context
+from typing import Any
+import json
+import os
+import subprocess
+import yaml
+
+
+def run(cmd: list[str]) -> str:
+	return subprocess.run(cmd, capture_output=True, encoding="utf8").stdout.strip()
+
+
+def beet_default(ctx: Context):
+	version = os.getenv("VERSION", "1.19")
+	prefix = int(os.getenv("PATCH_PREFIX", 0))
+
+	modules: list[dict[str, Any]] = [{"id": p.name} for p in sorted(ctx.directory.glob("gm4_*"))]
+
+	for module in modules:
+		id = module["id"]
+		try:
+			with open(f"{id}/beet.yaml", "r+") as f:
+				meta = yaml.safe_load(f)
+				module["name"] = meta.get("name", id)
+				module.update(meta.get("meta", {}).get("gm4", {}))
+		except:
```

Comment:
I think bare `except`s are really error-prone, I'd recommend looking for the specific exceptions here.